public/docs/ts/latest/glossary.jade

```diff
@@ -718,33 +695,34 @@ a#snake-case
 .l-sub-section
   :marked
     An expression is a !{_Lang}-like syntax that Angular evaluates within
-    a [data binding](#data-binding).  Learn how to write template expressions
-    in the [Template Syntax](!{docsLatest}/guide/template-syntax.html#template-expressions) chapter.
+    a [data binding](#data-binding). 
+
+    Read about how to write template expressions
+    in the [Template Syntax](!{docsLatest}/guide/template-syntax.html#template-expressions) page.
 
 :marked
   ## Transpile
 .l-sub-section
   :marked
     The process of transforming code written in one form of JavaScript
-    (e.g., TypeScript) into another form of JavaScript  (e.g., [ES5](#es5)).
+    (for example, TypeScript) into another form of JavaScript  (for example, [ES5](#es5)).
 
  :marked
   ## TypeScript
 .l-sub-section
   :marked
-    A version of JavaScript that supports most [ECMAScript](#ecmascript) 2015
-    language features and many features that may arrive in future versions
-    of JavaScript such as [Decorators](#decorator).
+    A version of JavaScript that supports most [ECMAScript 2015](#ecmascript=2015)
+    language features such as [Decorators](#decorator).
 
-    TypeScript is also noteable for its optional typing system which gives
-    us compile-time type-checking and strong tooling support (e.g. "intellisense",
+    TypeScript is also noteable for its optional typing system, which gives
+    us compile-time type checking and strong tooling support (for example, "intellisense",
     code completion, refactoring, and intelligent search). Many code editors
     and IDEs support TypeScript either natively or with plugins.
 
     TypeScript is the preferred language for Angular 2 development although
-    we are welcome to write in other JavaScript dialects such as [ES5](#es5).
+    you can use other JavaScript dialects such as [ES5](#es5).
 
-    Learn more about TypeScript on its [website](http://www.typescriptlang.org/).
+    Read more about TypeScript on its [website](http://www.typescriptlang.org/).
```

Comment:
"website" is too broad. Maybe just name the website:

Read more about TypeScript at [typescriptlang.org](http://www.typescriptlang.org/).